### ISSUE TITLE
Publish meeting-notes from December meeting

### ISSUE THREAD

**Issue by matthiaskoenig** [SED-ML/sed-ml.github.io#29]

**Comment by jonrkarr** [comment]
I don't think this is necessary. I don't think these details would be helpful to many people.

If we're motivated to share this, a simple solution would to post a link to the Google Documents with meeting notes.

**Comment by luciansmith** [comment]
Yes, that's exactly what this means.
